### PR TITLE
Fix WooCommerce Services JITM Stats

### PR DESCRIPTION
**Pull request by jeffstieler**
#### Changes proposed in this Pull Request:

* Rename `woocommerce_services` MC stat to `wooservices` (was hitting char limit)
* Don't bump activation stat as `launch` type, record as module activation

#### Testing instructions:

* Verify that view, dismissal, and activation stats are bumped using the `wooservices` group, and that `jitmActionToTake` is not `launch` for WooCommerce Services.

### CODE REVIEW

File: 3rd-party/woocommerce-services.php [hunk 1]
```diff
@@ -73,6 +73,9 @@ public function error_notice() {
 	 * @return bool result of installation/activation
 	 */
 	private function install() {
+		$jetpack = Jetpack::init();
+		$jetpack->stat( 'jitm', 'wooservices-activated-' . JETPACK__VERSION );
```
Comment (dereksmart) [inline 3rd-party/woocommerce-services.php]: Would it make sense to move this below after success, so we're not tracking failed attempts?

Comment (jeffstieler) [inline 3rd-party/woocommerce-services.php]: I'd rather track an additional stat to determine successful install.. as it sits, we should be able to glean that people are having installation problems by comparing the number of `activated` stats to active sites.. though that method isn't without it's flaws.

What do you think about tracking an `install` stat on button click, and an `activated` stat on successful result of the `install()` method?

Comment (dereksmart) [inline 3rd-party/woocommerce-services.php]: > What do you think about tracking an install stat on button click, and an activated stat on successful result of the install() method?

:shipit:

Comment (jeffstieler) [inline 3rd-party/woocommerce-services.php]: https://github.com/Automattic/jetpack/pull/6304/commits/60703b0d0be0232961569c2a9ba23edf8002f4cc

**Comment by dereksmart** [comment]
merged to 4.6 20adafb

**Pull request merged**